op-acceptance-tests/acceptance-tests.yaml

```diff
@@ -0,0 +1,15 @@
+# Configuration file for op-acceptor
+#
+# All acceptance tests need to be registered here for op-acceptor to run them.
+
```

Comment:
maybe add a comment to clarify that below are essentially  dummy tests for now?

Reply:
Okay, done.